### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by david63011**
here is the version of my desktop site built with media queries as well as HTML and CSS

### CODE REVIEW

**Review by Qoosim — CHANGES_REQUESTED** [review]
# **Status: Required Changes** :recycle: 

Hi @david63011, 

Good job so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Top Highlights

-  No linter errors. :heavy_check_mark: 
-   Adherence to Github Flow. :heavy_check_mark: 
-  Good commit messages. :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Good job for pushing it this far! :clap: . However, there are few things you still need to work on before your project can get approval.
     -  First, the margin right and left of your portfolio is not consistent. All the contents of your page is cut off on the right of your page. Kindly look into that and fix it. 

### Figma Design

figma-design-2


### Your Design

your-design-3

- [ ] Also, try to look into your placeholder texts in your input field, they are too closed to the left wall of your input field. Kindly give your placeholder texts some margin or padding left to move them a bit to the right.

### Figma Design

figma-design-1


### Your Design

your-design-1

- [ ] Try to follow the Figma Design as close as possible. This includes, font, font-size, font-family, font-weight, color, margin, padding, border, among others. Doing this would make our portfolio to look standard and professional.

### Figma Design

figma-btn


### Your Design

your-btn

-  Kindly remove the border from your `Get in touch` button.

- [ ]  I would also like to recommend you to look into the margin left and right of your list of projects section and give it equal margin left and right.

### Figma Design

figma-works


### Your Design

your-design-5

- [ ] Kindly add more information to README file and make it well organized. It does not look too professional with the current outlook. you can make it look better by following this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse

**Comment by david63011** [comment]
ok I fixed the margin problem on the right,  I fixed the placeholder text, I fixed the buttons, I fixed the margin on the project section, and I improved my readme

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @david63011, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

work2

 ## Highlights 💯 

- I appreciate the effort you put into your work ✔️ 

 ## Required Changes ♻️

- kindly advise you to follow the previous reviewer's advice.
- Kindly, it has quite a few differences from the original Figma design, I recommend you review your site thoroughly and compare fonts, images, colors, spaces between elements, and padding.

_Check the comments under the review._

## Optional suggestions

[OPTIONAL] Kindly, the initial file of your web page should always be index.html (not index2.html) I suggest you change it

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by david63011** [comment]
hi @mgmediaweb I am happy to make changes could you please be a little more specific i only have one code review left

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hello @david63011 👋,

_This is an additional review as a result of the second opinion request._

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._

## Required Changes ♻️
- [ ] As mentioned by the previous reviewer, there still a lot you have to do to make your app look more like the Figma design. 
  - The app does not respond well either in small and big screen.  You can learn more about css responsiveness [Here](https://www.w3schools.com/css/css_rwd_intro.asp)
  - There are so much overlapping, check out the screenshots below
 
| Original      | Yours |
  | ----------- | ----------- |
| Screenshot(2) | Screenshot (1) |

Kindly fix it.


_Check the comments under the review._


Keep rocking 🚀!

**Comment by david63011** [comment]
Ok I remade my entire middle section to make it more responsiveness  I hope that is enough

**Comment by david63011** [comment]
as well as change other projects with the desktop page

File: style2.css [hunk 1]
```diff
@@ -33,6 +116,21 @@ body {
...
+  margin-left: 30px;
+  color: #172b4d;
+}
+
+.para-box {
+  width: 1200px;
+  margin-left: 260px;
+}
+
+.paragraph2 {
```
Comment (ElsonOtake) [inline style2.css:147]: - [ ] Please correct the overlapping of text as required by the previous reviewer. Stick to the Figma design is a project requirement.
Your portfolio:
Screen Shot 2022-08-27 at 7 51 40 PM

Figma template:
Screen Shot 2022-08-27 at 7 51 59 PM

File: style2.css [hunk 2]
```diff
@@ -1157,15 +1015,37 @@ textarea {
     width: 369px;
   }
 
+  .type-of-cards {
```
Comment (ElsonOtake) [inline style2.css:1018]: - [ ] Please correct the overlapping as requested by the previous reviewer. Stick to the Figma template is a project requirement.
Your portfolio:
Screen Shot 2022-08-27 at 7 53 34 PM

Figma template:
Screen Shot 2022-08-27 at 7 54 53 PM

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @david63011, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥲 🥲 

woohoo-youre-almost

 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style2.css (hunk 2 above)
Comment (david63011) [reply style2.css:1018]: @ElsonOtake ok i have been trying all day i even rebuilt the whole site do you have any advise on how to do it

Comment (david63011) [reply style2.css:1018]: and now it is responsive I dont want next week to only resize the page

**Comment by david63011** [comment]
ok I fixed the overlapping of text,  I created more empty spaces on both sides (left and right) of your work sections, and i improved my readme

File: index2.html [hunk 3]
```diff
@@ -292,56 +304,56 @@ <h6 class="textline7">in touch</h6>
...
+                <input
+                  type="email"
+                  name="user_email"
+                  class="name-last-text-box"
+                  id="email"
+                  placeholder="Email Address"
+                  required=""
+                />
+              </li>
+              <li>
```
Comment (BenMukebo) [inline index2.html:338]: > * [ ]  I think it's better to remove the border of this button as requested by the previous reviewer. Stick to the Figma template is a project requirement.
>   Your portfolio:
> 
> Screen Shot 2022-08-27 at 7 09 47 PM
> 
> Figma template: Screen Shot 2022-08-27 at 7 10 39 PM

- [ ] Kindly address this 👍🏻

File: style2.css [hunk 4]
```diff
@@ -1157,15 +1015,37 @@ textarea {
...
+    flex-direction: column;
+    padding: 30px 0 0 0;
+    margin-bottom: 2em;
+    height: 352px;
+    width: 327px;
+    background: #ebf0ee;
+    border-radius: 5px;
+  }
+
   .grid-containers {
```
[the remark this answers is not in the document]
Comment (BenMukebo) [reply style2.css:1036]: - [ ] Great job so far, However you should make sure to center your work sections on the center of the page as requested by the previous reviewer, the margin-left should be legal to the margin-right

File: README.md [hunk 5]
```diff
@@ -1,10 +1,28 @@
...
 HTML and CSS
 
+**Live Demo**
+Coming Soon!
+
+**Getting Started**
+
+The first step in setting up this project in setting up your lintors. After that you can go to Figma and find a template and then use your HTML and CSS
+to design your site. Good luck!
+
```
[the remark this answers is not in the document]
Comment (BenMukebo) [reply README.md]: - [ ] Kindly address this 👍🏻

File: style2.css (hunk 1 above)
Comment (BenMukebo) [reply style2.css:161]: - [ ] Kindly address this 👍🏻

File: style2.css (hunk 2 above)
Comment (BenMukebo) [reply style2.css:1025]: - [ ] Kindly address this 👍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @david63011,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Please Instead of disabling `eslint`, it will be nice to fix all linter errors manually.

You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**